wallet/src/de/schildbach/wallet/ui/OnboardingActivity.kt

```diff
@@ -75,6 +75,9 @@ class OnboardingActivity : RestoreFromFileActivity() {
         viewModel = ViewModelProviders.of(this).get(OnboardingViewModel::class.java)
 
         walletApplication = (application as WalletApplication)
+
+        walletApplication.configuration.autoLogoutEnabled = false// for testing purpose
```

Comment:
should this be here if not for testing purposes

Reply:
As per brian comment in slack  while in testing mode disable the security so currently added this code